fog/sample-paykit/src/cached_tx_data/memo_handler.rs

```diff
@@ -63,6 +63,14 @@ impl MemoHandler {
         log::trace!(self.logger, "Obtained a memo: {:?}", memo_type);
         match memo_type.clone() {
             MemoType::Unused(_) => Ok(None),
+            MemoType::BurnRedemption(_) => {
```

Comment:
Any thoughts on this @garbageslam ?

Reply:
Looks reasonable 👍

Reply:
This is here so that the test client can check that RTH is working, and if it gets unexpected memo types it fails the test, so this all looks like it will work correctly to me.